### ISSUE TITLE
UPDATE: Update item from Single View page

### ISSUE THREAD

**Issue by ancasimon** [ancasimon/react-hoarder#14]
## User Story
As a user, when I click a update button on the Single Item page, I should be redirected to the edit page and should see a form pre-populated with all the information for the item I am editing. Once I make edits and push the save button, Firebase should edit and I should be redirected to the '/stuff' page.

## AC
WHEN I click Edit on an item on the Single Item View page,
THEN I go to an Edit Item page,
AND I see all the current values for the item's properties pre-populated,
AND I can change any of them,
AND when I click Update Item button on this form,
THEN the changes I made are saved,
AND I go to the My Stuff page where I can see the changes I just made.

## Dev Notes
* We are going to add Edit button and action on the Single View page - but we are relying on the edit feature already done for the Edit button from my Stuff page for the actual editing! So that needs to be done first.
* In the Single.js file:
1. Import the Link component from the react-router-dom package. 
1. Inside the return > inside the div > add a Link tag here for the Edit and give it button classes to make it look like a button. 
1. In the render, before the return, declare a variable that stores the itemId, which we can pull from params (`const { itemId } = this.props.match.params;`) and another variable that holds the path to the Edit page where we want to take the user when they click Edit here (` const editLink = `/edit/${itemId}`;`). 
1. Add the `to` attribute to the Link tag with the editLink object we just defined.

**Comment by ancasimon** [comment]
We did something similar to this for passing in multiple paths via a Link component: https://medium.com/@bopaiahmd.mca/how-to-pass-props-using-link-and-navlink-in-react-router-v4-75dc1d9507b4
